src/main/java/com/airlines/model/Crew.java

```diff
@@ -0,0 +1,42 @@
+package com.airlines.model;
+
+import java.util.List;
+
+public class Crew {
+    private int id;
+    private String name;
+    private List<CrewMember> crewMembers;
+
+    public Crew() {
+    }
+
+    public Crew(int id, String name, List<CrewMember> crewMembers) {
```

Comment:
it also should be immutable entity, so we need hide public no-args constructor and remove setters

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/1269dee2e4effc40beaddfca6d85b7799518548f